docs/source/concepts/SampleEnvironment.rst

```diff
@@ -94,9 +106,10 @@ Mantid will try the following approaches to find the path to the stl file (in or
 - Mantid will then check in the data search directories
 
 The stl file format doesn't natively support a scale so this should be specified
-in the scale attribute of the stilfile tag. Possible values are mm, cm or m.
+in the scale attribute of the stlfile tag. Possible values are mm, cm or m.
 
-Mantid does not provide tools to view meshes loaded from .stl files. There are however various free software tools available that can view .stl files:
+Stl mesh shapes can be plotted in Mantid, see :ref`Mesh_Plots`.
```

Comment:
I think there's a : missing after ref here which means the link isn't being displayed properly